### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by blakbox23**
In this project, I used an API with Covid related metrics and filtered out the data to show Covid numbers in Africa.  
Tasks in this project included:
- setting up react components,
- setting up routing,
- setting up redux store,
- fetching and filtering data from an API,
- styling components as per the requirements,
- Adding tests to the components as well as redux,,
- Deploying the webapp.
- Make a [Video presentation](https://www.loom.com/share/d580e56a8abf4a05bc933fb72cc67b98) of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,63 @@
...
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## Acknowledgments
 
-### Analyzing the Bundle Size
+- Hat tip to anyone whose code was used
+- Inspiration
+- Flat icons
+- [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
```
Comment (henatan99) [inline README.md]: - [x] I think the credit should go specifically to Nelson Sakwa the author of the original design, as required by the Creative Commons License. Kindly make a change here. :pray:

File: src/components/App.js [hunk 2]
```diff
@@ -5,9 +5,6 @@ import ItemDescription from './ItemDescription';
 import '../App.css';
```
Comment (henatan99) [inline src/components/App.js]: - [x] The `App` component is a top-level component. I recommend you take this file inside the `src` directory for a more logical document structure. :pray:

File: src/components/BodyList.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* eslint react/prop-types: 0 */import React, { useEffect } from 'react';
```
Comment (henatan99) [inline src/components/BodyList.js:1]: - [x] The `BodyList` component is a stateful component that deals with the redux store. So basically it is a `container` component. I recommend you create a `container` folder inside the `src` directory and move this file to the `container` folder for a more logical document structure. :pray:

File: src/components/ItemDescription.js [hunk 4]
```diff
@@ -8,15 +8,11 @@ function ItemDescription(props) {
   const country = props.country
```
Comment (henatan99) [inline src/components/ItemDescription.js:8]: - [x] The `ItemDescription` component is a stateful component that deals with the redux store. So basically it is a `container` component. I recommend you create a `container` folder inside the `src` directory and move this file to the `container` folder for a more logical document structure. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @blakbox23 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] The background colors of the cards on the Home page has a diagonal pattern in the original design. To be consistent with the original design kindly make changes accordingly. :pray: 

image

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by blakbox23** [comment]
Thanks @henatan99  👍🏾  I appreciate your review. I have made all the required changes

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**